packages/fast-components-react-base/src/utilities/extract-element.ts

```diff
@@ -5,7 +5,7 @@ import { isNil } from "lodash-es";
  * extracts the html element from React ref or simply returns html element passed in
  */
 export function extractHtmlElement(
-    sourceRef: React.RefObject<HTMLElement> | HTMLElement
+    sourceRef: React.RefObject<any> | HTMLElement
```

Comment:
what is necessitating this change?

Reply:
Because not all refs we'd want to extract a dom element from are necessarily of type HTMLElement.  In this case toolbar has a ref that points to a ToolbarItemGroup:

   private rootElement: React.RefObject<ToolbarItemGroup> = React.createRef<
        ToolbarItemGroup
    >();